cogni-libs/src/main/java/zone/cogni/libs/sparqlservice/impl/FusekiSparqlService.java

```diff
@@ -17,42 +27,65 @@
 import java.io.IOException;
 import java.io.StringWriter;
 import java.nio.charset.StandardCharsets;
+import java.util.Base64;
 import java.util.Collections;
 import java.util.function.Function;
 
 public class FusekiSparqlService implements SparqlService {
```

Comment:
why not use `ConfigurationProperties` so configuration is part of the component?

we could get rid of FusekiConfig this way.

Reply:
I'm not even sure where and how `SparqlServiceProvider` get executed.

Reply:
true
I propose we add that add as a future improvement. At same time when we do some clean up and deprecate one of the 2 sparql service versions.
Note, the advantage of the Provider is that we pass the type in the config. So the bean creation uses the provider and doesn't use a specific SparqlService class.

In mopo we have this as usage:
```  @Bean
  @MopoDataSource(MopoDataSource.Source.rdfstoreCuriaInput)
  public SparqlService sparqlServiceCuriaInput() {
    return sparqlServiceProvider().createSparqlService(MopoDataSource.Source.rdfstoreCuriaInput);
  }
```
  
  MopoDataSource.Source enum has 2 values (we use 2 different fuseki stores).